ddpui/celeryworkers/tasks.py

```
@@ -869,6 +907,96 @@
+                type=LogsSummarizationType.DEPLOYMENT,
+                flow_run_id=flow_run_id,
+                task_id=failed_task.get("id"),
+            )
+
+            logger.info(f"Triggered deployment summarization for task {failed_task.get('id')}")
+
+        logger.info(f"Successfully triggered summarization for flow_run {flow_run_id}")
+
+    except Exception as e:
+        logger.exception(f"Error triggering summarization for {flow_run_id}: {str(e)}")
+        raise self.retry(exc=e)
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)